column_buffer.go

```diff
@@ -2296,7 +2297,14 @@ func writeRowsFuncOfStruct(t reflect.Type, schema *Schema, path columnPath) writ
 			}
 		})
 
-		writeRows := writeRowsFuncOf(f.Type, schema, columnPath)
+		forEachStructTagOption(f, "parquet-value", func(_ reflect.Type, option, _ string) {
+			switch option {
+			case "list":
+				mapValuePath = []string{"list", "element"}
```

Comment:
There are conditions where the inner element is not called `element`, would it break this case?

Reply:
this is the write flow,
so we will always generate the file with `element`,
but it might cause an issue during the read if the `element` is missing.

do you maybe have an example of such case?